jetty-jmx/src/main/java/org/eclipse/jetty/jmx/MBeanContainer.java

```diff
@@ -293,4 +494,87 @@ private void unregister(ObjectName objectName)
             LOG.warn(x);
         }
     }
+
+    private class MetaData
+    {
+        final ManagedObject _managedObject;
+        final List<ManagedAttribute> _attributes = new ArrayList<>();
+        final List<Method> _getters = new ArrayList<>();
+        final List<ManagedOperation> _operations = new ArrayList<>();
+        final List<Method> _methods = new ArrayList<>();
+
+        MetaData(Class<?> oClass)
```

Comment:
Fields and constructor should be `private`.

Reply:
ack